components/eam/src/physics/crm/dummy_modules/debug_info.F90

```diff
@@ -0,0 +1,5 @@
+module debug_info
+!-------------------------------------------------------------------------------
+! Dummy module to override src/physics/cam/debug_info.F90
+!-------------------------------------------------------------------------------
+end module debug_info
```

Comment:
Why is github showing a red circle with a line through it at the end of the file?  Is it missing a newline at the end of the last line?

Reply:
yea, missing newline, I'll fix that.